GPT/gpt.py

```diff
@@ -192,20 +195,32 @@ def gpt_reformat_last(how_to_reformat: str):
             notify("No text to reformat")
             raise Exception("No text to reformat")
 
+    def gpt_paste_with_modifier(text: str, modifier: str = ""):
+        """Paste and apply modifications"""
+        match modifier:
+            case "snip":
+                actions.user.insert_snippet(text)
+            case _:
+                actions.user.paste(text)
+
     def gpt_insert_response(
-        result: str, method: str = "", cursorless_destination: Any = None
+        result: str,
+        modifier: str = "",
+        pasteDestination: str = "",
+        method: str = "",
+        cursorless_destination: Any = None,
     ):
         """Insert a GPT result in a specified way"""
-        match method:
+        match method or pasteDestination:
```

Comment:
this confuses me. Why are we switching on both these variables?

Reply:
I split the original talon list into two but didn't split the match statement. The match statement could be split too. Only one of these will be set though.